ios/RNOneginiSdk.swift

```diff
@@ -192,8 +192,14 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
     @objc
     func cancelRegistration(_ resolve: @escaping RCTPromiseResolveBlock,
                             rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
-        bridgeConnector.toRegistrationConnector.registrationHandler.cancelRegistration()
-        resolve(nil)
+        var canceled = bridgeConnector.toRegistrationConnector.registrationHandler.cancelBrowserRegistration()
+        canceled = bridgeConnector.toRegistrationConnector.registrationHandler.cancelPinCreation() || canceled
+        canceled = bridgeConnector.toRegistrationConnector.registrationHandler.cancelCustomRegistration() || canceled
+        if (canceled) {
```

Comment:
() not needed

Reply:
can you create a `let` for `bridgeConnector.toRegistrationConnector.registrationHandler` and use it in 3 places?